endpoints_management/control/wsgi.py

```diff
@@ -379,6 +386,12 @@ def _create_check_info(self, method_info, parsed_uri, environ):
 
         if api_key:
             api_key_valid = True
+        return (api_key_valid, api_key)
```

Comment:
Maybe: `return (bool(api_key), api_key)`, or let the caller do the `bool(api_key)`?

Reply:
Or does `api_key_valid` mean the API key has been verified?

Reply:
It appears to mean only that a non-`None` api key has been found. It should likely be cleaned up.